<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="graphicsView_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionKMeans_Algorithm"/>
    <addaction name="actionKohonen_s_Algorithm"/>
    <addaction name="actionNeural_Gas_Algorithm"/>
    <addaction name="separator"/>
    <addaction name="actionNeural_Gas_Image"/>
    <addaction name="actionExport_Image"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Centroids"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Data"/>
    <addaction name="actionLoad_image"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionKMeans_Algorithm">
   <property name="text">
    <string>KMeans Algorithm</string>
   </property>
  </action>
  <action name="actionKohonen_s_Algorithm">
   <property name="text">
    <string>Kohonen's Algorithm</string>
   </property>
  </action>
  <action name="actionNeural_Gas_Algorithm">
   <property name="text">
    <string>Neural Gas Algorithm</string>
   </property>
  </action>
  <action name="actionSet_Centroids">
   <property name="text">
    <string>Set Centroids</string>
   </property>
  </action>
  <action name="actionSet_Data">
   <property name="text">
    <string>Load data</string>
   </property>
  </action>
  <action name="actionLoad_image">
   <property name="text">
    <string>Load image</string>
   </property>
  </action>
  <action name="actionNeural_Gas_Image">
   <property name="text">
    <string>Neural Gas Image</string>
   </property>
  </action>
  <action name="actionExport_Image">
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
